src/support/SupportResourceType.ts

```diff
@@ -0,0 +1,18 @@
+import { FullyQualifiedResourceId } from 'src/libs/ajax/SamResources';
+import { ManagedGroupSummary } from 'src/support/ManagedGroupSummary';
+
+export interface ResourceTypeSummaryProps {
+  displayName: string;
+  fqResourceId: FullyQualifiedResourceId;
+}
+
+export interface SupportResourceType {
+  displayName: string;
+  resourceType: string;
+  detailComponent: (props: ResourceTypeSummaryProps) => React.ReactElement<any, any>;
```

Comment:
Alternatively, `ComponentType` (imported from react) might make it slightly clearer that this is a component?

```suggestion
  detailComponent: ComponentType<ResourceTypeSummaryProps>;
```